src/stores/Frame/FrameStore.ts

```diff
@@ -1222,9 +1223,9 @@ export class FrameStore {
 
         this.stokesFiles = [];
 
-        this.isPreview = false;
+        this.isPreview = frameInfo.preview;
```

Comment:
If preview is included in `this.frameInfo`, how about we change `this.isPreview` to a `@computed getter`?